packages/ERTP/src/displayInfo.js

```diff
@@ -61,7 +62,7 @@ export const coerceDisplayInfo = allegedDisplayInfo => {
     assert.equal(Reflect.ownKeys(allegedDisplayInfo).length, 0);
```

Comment:
I remember @erights saying that we are adding a property that might show up in something like `Reflect.ownKeys`. Was that for Far objects only?

Reply:
That's temporarily also for Data objects. Until this conversion is complete, at which point it's only Far.

Reply:
Ok, in that case, this check will break, I think.

Reply:
BTW in doing the Zoe conversion, I'm experimenting with the following change to `cleanProposal.js`, which has a similar issue:

```js

const rejectSymbols = record => {
  // Reject all symbols except for 'passStyle', which is used by the Data()
  // marker. When we remove Data, this should revert back to rejecting
  // absolutely all symbols.
  for (const sym of Object.getOwnPropertySymbols(record)) {
    if (sym !== PASS_STYLE) {
      assert.fail(X`no symbol properties allowed: ${q(String(sym))}`);
    }
  }
};


// Assert that the keys of `record` are all in `allowedKeys`. If a key
// of `record` is not in `allowedKeys`, throw an error. If a key in
// `allowedKeys` is not a key of record, we do not throw an error.
const assertKeysAllowed = (allowedKeys, record) => {
  const keys = Object.getOwnPropertyNames(record);
  assertSubset(allowedKeys, keys);
  rejectSymbols(record);
};

// similar change to `cleanKeywords()`
```

(which also depends upon a patch to marshal.js to expose the `PASS_STYLE` symbol)

Reply:
Actually based on discussion with @erights on keybase, I think this check is correct, as Far doesn't add extra properties to the object itself, but instead adds a new object in the inheritance chain.

Reply:
We just decided to change the way Data is represented to avoid this, in #2538. I won't land this PR until that fix has landed and I've reviewed the use of `ownKeys` in ERTP.